src/Mvc/Mvc.Core/test/Authorization/AuthorizeFilterTest.cs

```diff
@@ -513,26 +513,6 @@ public void CreateInstance_ReturnsNewFilterIfPolicyAndPolicyProviderAreNotSet(Au
             Assert.Same(policyProvider, actual.PolicyProvider);
         }
 
-        [Fact]
-        public async Task GetEffectivePolicyAsync_ReturnsCurrentPolicy_WhenNoEndpointMetadataIsAvailable()
```

Comment:
Is it worth a test that we don't cache? IE, we see the policy provider called twice. I'm thinking less about correctness and more about preventing an *optimization* in the future. Another idea would be a big scary comment.

Reply:
Yeah I saw a test that was verifying this already in the case of a policy provider in the past , I'll find it and copy and verify it for the normal code path as well